src/prompts/index.ts

```diff
@@ -0,0 +1,27 @@
+import { prompt } from "@/types";
+import { promptLayerGetPrompt, promptLayerPublishPrompt } from "@/utils";
+/**
+ * Get a prompt template from PromptLayer.
+ */
+const getPrompt = async (params: prompt.Retrieve): Promise<prompt.Response> => {
```

Comment:
Why are the types `prompt.Retrieve` or `prompt.Response`? Why is the folder called `prompt`?

It should be called `promptTemplate` if I understand correctly?

![CleanShot 2023-10-24 at 23 12 43](https://github.com/MagnivOrg/prompt-layer-js/assets/1629612/66bc897f-67f7-43be-abe4-4867f8428adb)

This tooltip isn't that descriptive to me when it's called "Response"

Reply:
I have updated to get types from a single types file. Will update the rest of the PRs as well.